featuretools/tests/entityset_tests/test_ww_es.py

```diff
@@ -286,11 +287,65 @@ def test_update_dataframe():
     assert es['table'].ww.schema == original_schema
 
 
-def test_add_last_time_indexes(es):
+def test_add_last_time_index(es):
     es.add_last_time_indexes(['products'])
 
     assert 'last_time_index' in es['products'].ww.metadata
 
+    assert es['products'].ww.metadata['last_time_index'] == 'last_time'
+    assert 'last_time' in es['products']
+    assert 'last_time_index' in es['products'].ww.semantic_tags['last_time']
+    assert isinstance(es['products'].ww.logical_types['last_time'], Datetime)
+
+
+def test_add_last_time_non_numeric_index(pd_es, ks_es, dask_es):
+    pd_es.add_last_time_indexes(['products'])
+    dask_es.add_last_time_indexes(['products'])
+    ks_es.add_last_time_indexes(['products'])
+
+    assert list(to_pandas(pd_es['products']['last_time']).sort_index()) == list(to_pandas(dask_es['products']['last_time']).sort_index())
+    assert list(to_pandas(pd_es['products']['last_time']).sort_index()) == list(to_pandas(ks_es['products']).sort_values('id')['last_time'])
+
+    assert pd_es['products'].ww.schema == dask_es['products'].ww.schema
+    assert pd_es['products'].ww.schema == ks_es['products'].ww.schema
+
+
+def test_lti_already_has_last_time_column(es):
+    col = es['customers'].ww.pop('loves_ice_cream')
+    col.name = 'last_time'
+
+    es['customers'].ww['last_time'] = col
+
+    assert 'last_time' in es['customers'].columns
+    assert isinstance(es['customers'].ww.logical_types['last_time'], Boolean)
+
+    es.add_last_time_indexes(['customers'])
+
+    # Current behavior will attempt to overwrite this column.
```

Comment:
That's a good point about the uniqueness of the name. My original thought processes around overwriting the column was to make sure that a user could call `add_last_time_indexes` twice without issues, but we probably want to minimize the chance that we overwrite an actual column. 

options to consider:
- `ft_last_time`
- `last_time_in_entityset`

Another option to avoid overwriting a user's column is to confirm that the column contains the `last_time_index` tag or that it's listed in the metadata as the last time index before proceeding. And raising an error telling users to rename otherwise

Reply:
Yeah, I think it makes sense to allow overwriting a column that featuretools has set. I like the idea of using the semantic tag for this purpose - couple that with a unique column name and I think that should make it highly unlikely that we would overwrite user data.

Could also do something like add an underscore to the column name to make it less likely to be something already present up `_ft_last_time`, for example.

Reply:
Added the column name change and semantic tag check